android/sound.cpp

```diff
@@ -195,17 +179,15 @@ int CSound::Init ( const int iNewPrefMonoBufferSize )
 
     // create memory for intermediate audio buffer
     vecsTmpInputAudioSndCrdStereo.Init ( iOboeBufferSizeStereo );
-    mOutBuffer.reset ( iOboeBufferSizeStereo * RING_FACTOR );
+    mOutBuffer.Init ( iOboeBufferSizeStereo * RING_FACTOR );
 
     return iOboeBufferSizeMono;
 }
 
 // This is the main callback method for when an audio stream is ready to publish data to an output stream
 // or has received data on an input stream. As per manual much be very careful not to do anything in this back that
 // can cause delays such as sleeping, file processing, allocate memory, etc.
-oboe::DataCallbackResult CSound::onAudioReady ( oboe::AudioStream* oboeStream,
```

Comment:
And I guess this is one of the things we may have to live with going with the .clang-format.